solarforecastarbiter/metrics/tests/test_preprocessing.py

```diff
@@ -342,6 +342,47 @@ def test_resample_and_align_ref_error_prob(prob_forecasts, single_observation):
             fx_obs, fx_series, obs_series, ref_series, tz)
 
 
+def test_resample_and_align_prob(prob_forecasts, single_observation):
+    tz = 'UTC'
+    fx_obs = datamodel.ForecastObservation(
+        prob_forecasts,
+        single_observation)
+    fx_series = THREE_HOUR_SERIES
+    obs_series = THREE_HOUR_SERIES
+    preprocessing.resample_and_align(
+        fx_obs, fx_series, obs_series, None, tz)
+    fx_values, obs_values, ref_values, _ = preprocessing.resample_and_align(
+        fx_obs, fx_series, obs_series, None, tz)
+    pd.testing.assert_index_equal(fx_values.index,
```

Comment:
and I think we should be using `assert_frame_equal` here.

Reply:
I can do that but need to convert the observation to dataframe. This only will work because the forecast has only a single constant value of course

Reply:
my bad. My intention was to suggest a test that asserts that `fx_values` is what it's supposed to be. Good to keep the `assert_index_equal` in addition to that.

Reply:
Ok. Changed them back.